dev/cnf/oss_source_dependencies.maven

```diff
@@ -19,6 +19,7 @@ commons-discovery:commons-discovery:0.2
 commons-httpclient:commons-httpclient:3.1
 commons-io:commons-io:2.6
 commons-io:commons-io:2.8.0
+commons-io:commons-io:2.11.0
```

Comment:
Did you do the source dependencies approach?  If not, then you should remove 2.8.0 and 2.11.0 from this file.